app/data/mwss_4.json

```diff
@@ -0,0 +1,3729 @@
+                                "id" : "weekly-pay-block-14",
+                                "when": [
+                                    {
+                                        "id" : "weekly-pay-answer-12-0",
+                                        "condition": "equals",
+                                        "value":"No"
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                },
+                {
+                    "id": "weekly-pay-block-13",
+                    "sections": [
+                        {
+                            "description": "<p>1 December 2016 to 31 December 2016</p>",
+                            "id": "weekly-pay-section-13",
+                            "number": "2",
+                            "questions": [
+                                {
+                                    "answers": [
+                                        {
+                                            "description": "",
+                                            "guidance": "",
+                                            "id": "weekly-pay-answer-13-0",
+                                            "label": "Percentage increase of new pay rates",
+                                            "mandatory": true,
+                                            "options": [],
+                                            "q_code": "100",
+                                            "type": "TextArea"
+                                        },
+                                        {
+                                            "description": "Please use the format dd/mm/yyyy",
+                                            "guidance": "",
+                                            "id": "weekly-pay-answer-13-1",
+                                            "label": "If your figures are back dated, from when:",
+                                            "mandatory": true,
+                                            "options": [],
+                                            "q_code": "110",
+                                            "type": "TextArea"
+                                        },
+                                        {
+                                            "description": "Please enter the percentage or number  of staff affected",
+                                            "guidance": "",
+                                            "id": "weekly-pay-answer-13-2",
+                                            "label": "What was the percentage or number of weekly paid employees who received this new pay rate?",
+                                            "mandatory": true,
+                                            "options": [],
+                                            "q_code": "120",
+                                            "type": "TextArea"
```

Comment:
* This should be a numeric input
* The text `Please enter the percentage or number  of staff affected` is missing